js/Course.js

```diff
@@ -11,11 +11,12 @@
  * @constructor
  * @author Bradley Cai
  */
-function Course(quarter, name, nameID, gt, units,
+function Course(quarter, year, name, nameID, gt, units,
                 days, hour1, min1, hour2, min2, bldg, room) {
 
     //General information
-    this.quarter = (quarter === "" ? "None provided" : quarter); //written as a string in the format of "season####"
+    this.quarter = quarter;
```

Comment:
It was easier just to leave it blank for checking if it's empty later on. Since the quarter never really gets shown to the user (at least not yet) and since I check if the quarter is empty often, i decided to just leave it blank if none was provided.